src/bundle/Resources/translations/ezplatform_content_forms_policies.en.xliff

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:jms="urn:jms:translation" version="1.2">
+  <file source-language="en" target-language="en" datatype="plaintext" original="not.available">
+    <header>
+      <tool tool-id="JMSTranslationBundle" tool-name="JMSTranslationBundle" tool-version="1.1.0-DEV"/>
+      <note>The source node in most cases contains the sample message as written by the developer. If it looks like a dot-delimitted string such as "form.label.firstname", then the developer has not provided a default message.</note>
+    </header>
+    <body>
+      <trans-unit id="3742f2cc514848bd4a5eb03430e412d44f5d9ec2" resname="policy.limitation.identifier.class">
+        <source>Class</source>
```

Comment:
I believe all the labels from this file are used for the same cases as in https://github.com/ezsystems/ezplatform-admin-ui/blob/master/src/bundle/Resources/translations/ezplatform_content_forms_policies.en.xliff, so they should be the same

Reply:
> I believe all the labels from this file are used for the same cases as in https://github.com/ezsystems/ezplatform-admin-ui/blob/master/src/bundle/Resources/translations/ezplatform_content_forms_policies.en.xliff, so they should be the same

I actually don't think `ezplatform_content_forms_policies.en.xliff` belongs in this package.